terragrunt/aws/s3/s3.tf

```
@@ -0,0 +1,43 @@
+module "s3_buckets" {
+  source      = "github.com/cds-snc/terraform-modules?ref=v5.1.8//S3"
+  for_each    = var.buckets
+  bucket_name = each.key
+
+  logging = {
+    target_bucket = module.log_bucket.s3_bucket_id
+  }
+
+  billing_tag_value = var.billing_code
+}
+
+module "log_bucket" {
+  source            = "github.com/cds-snc/terraform-modules?ref=v5.1.8//S3_log_bucket"
+  bucket_name       = "${var.product_name}-${var.env}-logs"
```

Totally missed this one - could you please change this to be something like:
```suggestion
  bucket_name       = "cds-website-s3-bucket-logs"
```